--- cairo-dock-plugins-2.4.0~2/CMakeLists.txt.dbusmenu~	2012-02-17 01:26:10.000000000 +0100
+++ cairo-dock-plugins-2.4.0~2/CMakeLists.txt	2012-02-17 01:26:39.067459919 +0100
@@ -491,18 +491,12 @@ endif()
 message (STATUS "> Indicator-Applet:")
 # Note: the names of dbusmenu-glib, dbusmenu-gtk and indicator have changed...
 pkg_check_modules (DBUSMENU_04 QUIET dbusmenu-glib-0.4)
-pkg_check_modules (DBUSMENU_GTK_04 QUIET dbusmenu-gtk-0.4)
 pkg_check_modules (INDICATOR_04 QUIET indicator-0.4)
 if ("${DBUSMENU_04_FOUND}" STREQUAL "1")
 	pkg_check_modules (DBUSMENU dbusmenu-glib-0.4)
 else()
 	pkg_check_modules (DBUSMENU dbusmenu-glib)
 endif()
-if ("${DBUSMENU_GTK_04_FOUND}" STREQUAL "1")
-	pkg_check_modules (DBUSMENU_GTK dbusmenu-gtk-0.4)
-else()
-	pkg_check_modules (DBUSMENU_GTK dbusmenu-gtk)
-endif()
 if ("${INDICATOR_04_FOUND}" STREQUAL "1")
 	pkg_check_modules (INDICATOR_APPLET indicator-0.4 libido-0.1)
 	set (INDICATOR_APPLICATIONADDED_HAS_HINT 1) # now the ApplicationAdded signal (Status-Notifier) has a new parameter (hint) => https://code.launchpad.net/~ted/indicator-application/name-hints/+merge/67213
@@ -511,7 +505,7 @@ else()
 	set (INDICATOR_APPLICATIONADDED_HAS_HINT 0)
 endif()
 
-if ("${INDICATOR_APPLET_FOUND}" STREQUAL "1" AND "${DBUSMENU_FOUND}" STREQUAL "1" AND "${DBUSMENU_GTK_FOUND}" STREQUAL "1")
+if ("${INDICATOR_APPLET_FOUND}" STREQUAL "1" AND "${DBUSMENU_FOUND}" STREQUAL "1" AND "${DBUSMENU_FOUND}" STREQUAL "1")
 	#message (STATUS "DBUSMENU_VERSION : ${DBUSMENU_VERSION}")
 
 	STRING (REGEX REPLACE "\\..*" "" DBUSMENU_MAJOR "${DBUSMENU_VERSION}")
@@ -538,8 +532,8 @@ if ("${INDICATOR_APPLET_FOUND}" STREQUAL
 	add_subdirectory (Indicator-applet)
 	set (with_indicator "yes")
 else()
-	message (WARNING "These modules are required to compile Indicators applet (MeMenu and Messaging-Menu): indicator(-0.4), libido-0.1, dbusmenu-glib(-0.4), dbusmenu-gtk(-0.4)")
-	set (MODULES_MISSING "${MODULES_MISSING} indicator(-0.4) libido-0.1 dbusmenu-glib(-0.4) dbusmenu-gtk(-0.4)")
+	message (WARNING "These modules are required to compile Indicators applet (MeMenu and Messaging-Menu): indicator(-0.4), libido-0.1, dbusmenu-glib(-0.4)")
+	set (MODULES_MISSING "${MODULES_MISSING} indicator(-0.4) libido-0.1 dbusmenu-glib(-0.4)")
 endif()
 
 ############# ILLUSION #################
@@ -971,10 +965,10 @@ add_subdirectory (stack)
 
 ############# STATUS NOTIFIER #################
 message (STATUS "> Status Notifier:")
-if ("${DBUSMENU_GTK_FOUND}" STREQUAL "")
+if ("${DBUSMENU_FOUND}" STREQUAL "")
 	message (STATUS "Could not find Status-Notifier libs; Cairo-Dock won't be built with Status-Notifier support.")
-	message (WARNING "This module is required to compile Status Notifier applet: dbusmenu-gtk(-0.4)")
-	# set (MODULES_MISSING "${MODULES_MISSING} dbusmenu-gtk(-0.4)") # it should already be in the list.
+	message (WARNING "This module is required to compile Status Notifier applet: dbusmenu-lib(-0.4)")
+	# set (MODULES_MISSING "${MODULES_MISSING} dbusmenu-gtk(0.4)") # it should already be in the list.
 	set (with_status_notifier "no")
 else()
 	if ("${force_indicator_applicationadded_have_hint}" STREQUAL "yes")
--- cairo-dock-plugins-2.4.0~2/Status-Notifier/src/applet-struct.h.dbusmenu~	2011-10-03 16:51:28.000000000 +0200
+++ cairo-dock-plugins-2.4.0~2/Status-Notifier/src/applet-struct.h	2012-02-17 01:26:10.730436103 +0100
@@ -21,8 +21,8 @@
 #ifndef __CD_APPLET_STRUCT__
 #define  __CD_APPLET_STRUCT__
 
-#include <libdbusmenu-gtk/menuitem.h>
-#include <libdbusmenu-gtk/menu.h>
+#include <libdbusmenu-gtk3/menuitem.h>
+#include <libdbusmenu-gtk3/menu.h>
 
 #include <cairo-dock.h>
 
--- cairo-dock-plugins-2.4.0~2/Status-Notifier/src/CMakeLists.txt.dbusmenu~	2011-10-03 16:51:28.000000000 +0200
+++ cairo-dock-plugins-2.4.0~2/Status-Notifier/src/CMakeLists.txt	2012-02-17 01:26:10.730436103 +0100
@@ -30,15 +30,15 @@ add_definitions (-DCD_PLUGINS_DIR="${plu
 
 include_directories (
 	${PACKAGE_INCLUDE_DIRS}
-	${DBUSMENU_GTK_INCLUDE_DIRS})
+	${DBUSMENU_04_INCLUDE_DIRS})
 
 link_directories (
 	${PACKAGE_LIBRARY_DIRS}
-	${DBUSMENU_GTK_LIBRARY_DIRS})
+	${DBUSMENU_04_LIBRARY_DIRS})
 
 target_link_libraries (${PACKAGE_STATUS_NOTIFIER}
 	${PACKAGE_LIBRARIES}
-	${DBUSMENU_GTK_LIBRARIES})
+	${DBUSMENU_04_LIBRARIES})
 
 ########### install files ###############
 
