--- cairo-dock-plugins-2.4.0~2/CMakeLists.txt~	2011-10-03 16:51:28.000000000 +0200
+++ cairo-dock-plugins-2.4.0~2/CMakeLists.txt	2012-02-17 01:02:58.109275866 +0100
@@ -230,7 +230,7 @@ if ("${RUBY_LIB_DIR}" STREQUAL "" OR "${
 else()
 	string (REGEX REPLACE "lib/ruby/[0-9].[0-9]" "" RUBY_LIB_DIR_INSTALL "${RUBY_LIB_DIR}")
 	string (REGEX REPLACE "${RUBY_LIB_DIR_INSTALL}" "" RUBY_LIB_DIR_INSTALL "${RUBY_LIB_DIR}")
-	set (RUBY_LIB_DIR "${CMAKE_INSTALL_PREFIX}/${RUBY_LIB_DIR_INSTALL}")
+	set (RUBY_LIB_DIR "${RUBY_LIB_DIR_INSTALL}")
 	message (STATUS "   will be installed in: ${RUBY_LIB_DIR}")
 	set (RUBY_FOUND "TRUE")
 	set (with_ruby "yes")
